Updating wording for contacting support (#1045)
codatio/codat-docs: docs/integrations/accounting/myob/myob-dev-registration.md

------------------------

> If you have any questions on the information shared above, please raise a ticket with our support team through [this form](https://codat.zendesk.com/hc/en-gb/requests/new).

Reviewer: ```suggestion
If you have any questions on the information shared above, please use our [support request form](https://codat.zendesk.com/hc/en-gb/requests/new) to raise a ticket.
```
